### ISSUE TITLE
Internal server error 500 after fresh install on new Debian 8 system

### ISSUE THREAD

**Issue by souldessin** [modoboa/modoboa-installer#31]
I have a virtual machine that I just installed a fresh version of Debian 8 on and the first thing that I did was run the Modoboa-Installer. When it is done, I just get a white page that says "internal server error."

The only error log that show anything other than the access log for nginx is the uwsgi log in /var/log/uwsgi/app/modoboa_instance.log here:

`Thu Apr 14 21:57:16 2016 - unable to load app 0 (mountpoint='mercury.eastfountain.com|') (callable not found or import error)
Thu Apr 14 21:57:16 2016 - --- no python application found, check your startup logs for errors ---
mercury.eastfountain.com [pid: 1050|app: -1|req: -1/1] 192.168.1.178 () {42 vars in 681 bytes} [Thu Apr 14 21:57:16 2016] GET / => generated 21 bytes in 11 msecs (HTTP/1.1 500) 2 headers in 83 bytes (1 switches on core 0)
Traceback (most recent call last):
  File "./instance/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/core/wsgi.py", line 14, in get_wsgi_application
    django.setup()
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/apps/registry.py", line 78, in populate
    raise RuntimeError("populate() isn't reentrant")
RuntimeError: populate() isn't reentrant
`
I have checked online for anything related and the installer did install everything that is called for.

Thank you for your help. I have reloaded this 4 different times thinking that I did something wrong and I can't get more sterile than this last load.
I have

**Comment by tonioo** [comment]
@souldessin Sorry for the late answer. Have you tried again recently ?

**Comment by pengc99** [comment]
Just wanted to add that I'm having the same problem. Same error. Installed with latest version pulled with git.

**Comment by pengc99** [comment]
I found that restarting nginx will correct the problem

**Comment by adminsg** [comment]
in my case a "service uwsgi restart" fix the problem

Sometimes it happens to me when i do a reboot of my system

**Comment by tonioo** [comment]
Looks like this issue is related to Django itself:
https://code.djangoproject.com/ticket/24523

**Comment by souldessin** [comment]
What would you recommend that I use to get this running? Should I use a different flavor of Linux or version of Debian? Or should I try installing this without the installer and on Apache?

**Comment by tonioo** [comment]
I'll try to find a fix. Can you tell me what are the extensions you use?

**Comment by souldessin** [comment]
Ok, I have figured it out. First off I have to apologize. The real problem was on my end with configurations, not in Modoboa. The root of the issue was that I setup the server to handle Modoboa but I was not able to setup the DNS for the address that this server was to use and had it on a temporary one with plans to have the primary URL take over. Modoboa was serving the 500 error for all other URLs that went to the server. I thought that I had added the temporary server address to the accepted URLs but I clearly had not done so in the right config file. I was finally able, in the past week, to re-setup the server with everything as it should be and Modoboa installed without issue and is running fine. I have been testing it for the week without issue.

tl:dr - The server address entered during setup needs to be working and pointed to the server. I could not find documentation on how to add another accepted URL to Modoboa.
